Cargo.toml

```diff
@@ -15,13 +15,17 @@ panic = "abort"
 lto = true
 
 [features]
-default = ["log-serial", "log-panic"]
+default = ["log-serial", "log-panic", "ram32", "rom"]
 # Have the log! macro write to serial output. Disabling this significantly
 # reduces code size, but makes debugging essentially impossible
 log-serial = []
 # Log panics to serial output. Disabling this (without disabling log-serial)
 # gets you most of the code size reduction, without losing _all_ debugging.
 log-panic = ["log-serial"]
+# Support launching the firmware from 32-bit unpaged protected mode.
```

Comment:
I don't have an opinion on this either way. I think @bonzini was saying that with PVH it starts in protected mode so maybe this would be useful for that case?

Reply:
Yeah, I suppose you could have a PVH feature that requires ram32 and adds the PVH ELF note. Multiboot too.

Reply:
This is no longer relevant, the PVH code in #26 has all the RAM code.